warehouse/core/src/main/java/datawave/ingest/protobuf/TermWeight.java

```diff
@@ -125,7 +125,7 @@ public Info getDefaultInstanceForType() {
             return defaultInstance;
         }
 
-        private final com.google.protobuf.UnknownFieldSet unknownFields;
+        private final transient com.google.protobuf.UnknownFieldSet unknownFields;
```

Comment:
Won't this change get reverted when it is regenerated?

Reply:
reverted